src/vtbackend/Functions_test.cpp

```diff
@@ -17,45 +17,74 @@ inline std::ostream& operator<<(std::ostream& os, FunctionDefinition const& f)
 using namespace std;
 using namespace terminal;
 
+SupportedSequences seq;
```

Comment:
I personally think that globals should be used only if we have to. I know that this will add a little bit of code repetition, but it's a trivial line to add to the test function bodies. :)

Reply:
The name `seq` is not really self explaining. Maybe `sequences` or `availableSequences` or similar?